cardano-node/src/Cardano/Tracing/OrphanInstances/Network.hs

```diff
@@ -712,12 +709,16 @@ instance ToObject SlotNo where
     mkObject [ "kind" .= String "SlotNo"
              , "slot" .= toJSON (unSlotNo slot) ]
 
-
-instance ConvertRawHash header => ToObject (TraceFetchClientState header) where
+instance (HasHeader header, ConvertRawHash header)
+ => ToObject (TraceFetchClientState header) where
```

Comment:
Please don't mix refactoring and essential changes; this makes reviews / rebases harder.

Reply:
It was either that, or code duplication.

Reply:
@coot, addressed this.

Reply:
Thanks